libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -451,6 +456,82 @@ defaultSelfDeletingMessagesStatus =
     TeamFeatureEnabled
     (TeamFeatureSelfDeletingMessagesConfig 0)
 
+----------------------------------------------------------------------
+-- PaymentStatus
+
+data PaymentStatusUpdate = Lock | Unlock
```

Comment:
Why don't you use `PaymentStatus` for this?  Then you could also use `FromByteString` to implement the `FromHttpApiData` instance.

Reply:
Done